lib/Byng/Pimcore/Sitemap/SitemapPlugin.php

```diff
@@ -16,6 +16,7 @@
 namespace Byng\Pimcore\Sitemap;
 
 use Pimcore\API\Plugin as PluginLib;
+use Pimcore\Model\Property\Predefined;
```

Comment:
perhaps import this as PredefinedProperty? It'll just make it easier to see what it is, like you did below?